internal/config.go

```diff
@@ -26,6 +28,43 @@ type ConfigCORS struct {
 	AllowCredentials bool     `yaml:"allow_credentials"`
 }
 
+// ConfigProxy is a representation of section proxy of the yaml
+type ConfigProxy struct {
+	Url  string    `yaml:"url"`
+	Mode ProxyMode `yaml:"mode"`
+}
+
+// ProxyMode is enumeration of proxy server modes
+type ProxyMode int
```

Comment:
```suggestion
type ProxyMode uint8
```

Reply:
Why do we need specifically uint8?

Reply:
we prefer declare the enums with a more accurate type possible as standard, 8bits is more than enough to a enum and in most case u don't need negative numbers, so `uint8` is a elegant solution